R/hooks.R

```diff
@@ -81,6 +82,21 @@ log_errors <- function(muffle = getOption("logger_muffle_errors", FALSE)) {
     globalCallingHandlers(
       error = structure(function(m) {
         logger::log_level(logger::ERROR, m$message, .topcall = m$call)
+        if (traceback) {
+          bt <- .traceback(3L)
+          logger::log_level(logger::ERROR, "Traceback:", .topcall = m$call)
+          for (i in seq_along(bt)) {
+            msg <- paste0(length(bt) - i + 1L, ": ", bt[[i]])
+            ref <- attr(bt[[i]], "srcref")
+            file <- attr(ref, "srcfile")
+            if (inherits(file, "srcfile")) {
+              file <- basename(file$filename)
+              line <- paste(unique(c(ref[1L], ref[3L])), collapse = "-")
+              msg <- paste0(msg, " at ", file, " #", line)
+            }
+            logger::log_level(logger::ERROR, msg, .topcall = m$call)
```

Comment:
`msg` is very likely to include some special chars in the traceback to blow up the default `formatter_glue`, and interpolation is not needed anyway, so I think skipping the formatter is preferred here:

```suggestion
            logger::log_level(logger::ERROR, skip_formatter(msg), .topcall = m$call)
```

Reply:
Good call